app/images/icons/Makefile

```diff
@@ -1,5 +1,5 @@
 BROWSER_SIZES := 16 24 32 48 64
-ANDROID_SIZES := 48 72 96 144 192
+ANDROID_SIZES := 72 96 144 192
```

Comment:
Does Android want 48×48, though? I think it would be best to encode this fact here and deal with the duplication elsewhere. Otherwise, this will look like 48×48 isn't used on Android and might get removed.